src/components/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { forwardRef } from "react";
+import {
+  chakra,
+  Tooltip as ChakraTooltip,
+  useMultiStyleConfig,
+} from "@chakra-ui/react";
+
+export interface TooltipProps {
+  /** Any child node passed to the component. */
+  children: React.ReactNode;
+  /** Value used to populate the tooltip content. */
+  content: string | React.ReactNode;
+  /** A class name for the TextInput parent div. */
+  className?: string;
+  /** ID that other components can cross reference for accessibility purposes. */
+  id?: string;
+  /** Adds the `disabled` and `aria-disabled` prop to the input when true. */
+  isDisabled?: boolean;
+}
+
+export const Tooltip = chakra(
+  forwardRef<HTMLDivElement, TooltipProps>((props, ref?) => {
+    const { children, className, content, id, isDisabled, ...rest } = props;
+
+    const styles = useMultiStyleConfig("Tooltip", {});
+
+    return (
+      <ChakraTooltip
+        hasArrow
+        aria-label={typeof content !== "string" ? "Tooltip" : undefined}
```

Comment:
I think this might need to be `content` when it is a string instead of `undefined`. `"Tooltip"` is a bit too generic but I don't have any better suggestions right now.

Reply:
Correct me if I'm wrong here: aria-labels are used to describe elements that are not already defined through text, so the tooltip content should serve as definition. Otherwise, the aria-label will be redundant with the given string.

Reply:
Yes, you're right. Let's say you're viewing a page with a screenreader and there are multiple `Tooltip`s on the page that render icons or images. In that case, all the tooltips will say "Tooltip" and will be confusing for users. I'm not saying this is wrong, it can just be improved but I don't know how. We can consider this as an issue at a later time, so let's leave this for now.